src/app/scenarios/Scenario.service.test.js

```diff
@@ -115,4 +125,20 @@ describe('ScenarioService', () => {
             });
         });
     });
+
+    describe('Delete scenarios', () => {
+        it('should be defined', () => {
```

Comment:
this test is useless cause next test will fail in any case if there will be no "removeScenario" method. !fix